plugins/wasmedge_zlib/zlibfunc.h

```diff
@@ -0,0 +1,694 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+#pragma once
+
+#include "zlibbase.h"
+
+namespace WasmEdge {
+namespace Host {
+
+/*
+class WasmEdgeZlibZlibVersion : public WasmEdgeZlib<WasmEdgeZlibZlibVersion> {
+public:
+  WasmEdgeZlibZlibVersion(WasmEdgeZlibEnvironment &HostEnv)
+      : WasmEdgeZlib(HostEnv) {}
+  Expect<uint32_t> body(const Runtime::CallingFrame &Frame);
+};
+*/
```

Comment:
Remove this.

Reply:
I kept these as reminders, that these classes will later be uncommented, when their implementation is done.